docs/reference/mount-write-mode-limitations.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Mount Write Mode Limitations
+description: Limitations of the mount write mode
+grand_parent: Features
+parent: Mount
+---
+
+# Mount Write Mode Limitations
+
+## OS Limitations
+- Mount write mode is currently supported only on macOS.
+
+## Everest Partial-Read Mode
+- Mount write mode is not supported with Everest partial-read mode.
+
```

Comment:
Why does it need it's own title? There's no much to say here, also partial-read mode not released yet. 
**I wouldn't mention this at all.**